docs/appendices/release-notes/5.8.4.rst

```diff
@@ -92,23 +85,27 @@ Fixes
   operator on :ref:`REAL <type-real>` array columns against literals to return
   invalid results.
 
-- Fixed an issue that caused :ref:`in <sql_in_array_comparison>` operator with
-  array typed column on left hand side of the arguments to return invalid
-  results.
-
 - Fixed an issue that caused ``=`` or ``!=`` :ref:`ANY <sql_any_array_comparison>`
   operator to throw a ``ClassCastException`` when the right hand side argument
   was more than 1 dimensions higher than the left hand side argument.
 
+- Fixed an issue that caused :ref:`ANY <sql_any_array_comparison>` operator to
+  throw a ``ClassCastException`` when the arguments were nested arrays.
+
 - Fixed an issue that caused cast to :ref:`NUMERIC <type-numeric>` to return
   incorrect results, with missing ``scale``. This issue has only affected
   multi-node cluster installations.
 
 - Fixed an integer overflow issue when the total shards (shards and replicas)
   configured for a table are greater than ``Integer.MAX_VALUE``.
 
-- Fixed an issue that caused :ref:`ANY <sql_any_array_comparison>` operator to
-  throw a ``ClassCastException`` when the arguments were nested arrays.
+- Fixed an issue that caused ``=`` :ref:`ANY <sql_any_array_comparison>`
+  operator to throw a ``ClassCastException`` when the right hand side argument
+  was more than 1 dimensions higher than the left hand side argument.
```

Comment:
Why extra change? We already have this above in https://github.com/crate/crate/pull/16719/files#diff-9e29b7d6200ded69005f0ee466be11208936dacb4c68b642ff0b9a6f4c5e93d3R88

Reply:
This is the file from the 5.8 branch

Reply:
Looks like there is difference between 5.8 and master on this file.

Reply:
I would use the 5.8 file.

Reply:
Looks like 5.8 have an extra entry which we don't have on master. 



On master we have a single entry for `=` and `!=`
```
- Fixed an issue that caused ``=`` or ``!=`` :ref:`ANY <sql_any_array_comparison>`
  operator to throw a ``ClassCastException`` when the right hand side argument
  was more than 1 dimensions higher than the left hand side argument.
```

On 5.8 we have entry with 2 operator and also  `"= " only` entry, added in https://github.com/crate/crate/commit/eeb2bc59d9fb81e5b2a6bf5cf41c66b9aef6a4f2#diff-9e29b7d6200ded69005f0ee466be11208936dacb4c68b642ff0b9a6f4c5e93d3R74

I'm not sure if we should instead clean up this entry from the 5.8 branch.

@jeeminso could you please take a look?

Reply:
They should def. be the same file.

Reply:
Sorry it looks like I made an error while resolving conflicts in release notes.

Reply:
As Baur noted https://github.com/crate/crate/pull/16719/files#diff-9e29b7d6200ded69005f0ee466be11208936dacb4c68b642ff0b9a6f4c5e93d3R88 should replace this.

Reply:
I will fix this, it will be easier.

Reply:
The remaining diffs are just ordering changes, @matriv could you review this?

Reply:
done

Reply:
ok, so the version in master is how it is suppose to be and 5.8 is not correct?

Reply:
Yes, but 5.8 is fixed as well https://github.com/crate/crate/pull/16721